### DISCUSSION TITLE
"JUST DRIVE" mode from PSP version

### DISCUSSION THREAD

**Discussion by Sonic-TV** [emoose/OutRun2006Tweaks#228] [Suggestions]
### Describe the feature you'd like to see added

This should help those casual players to start a outrun game without headaches.

**Comment by Partoleita123** [comment]
Adding a way to outrun random people or friends wirelessly but within the PSP itself would be really cool.

**Comment by Partoleita123** [comment]
Adding a way to outrun random people or friends wirelessly but within the PSP itself would be really cool.
